### ISSUE TITLE
Clarify why you need to disable SCL to -1 transport rules when uring Enhanced filtering?

### ISSUE THREAD

**Issue by JonesMikael** [MicrosoftDocs/OfficeDocs-Exchange#3156]
Can you please clarify as to WHY and if it's REQUIRED we need to perform the following if you enable Enhanced filtering? Isn't it a valid request to user Enhanced filtering but still perform SCL=-1?

> If you have mail flow rules (also known as transport rules) that set the [SCL to -1](https://docs.microsoft.com/en-us/exchange/security-and-compliance/mail-flow-rules/use-rules-to-set-scl) for messages that flow through this connector, you must [disable those mail flow rules](https://docs.microsoft.com/en-us/exchange/security-and-compliance/mail-flow-rules/manage-mail-flow-rules#enable-or-disable-a-mail-flow-rule) after you enable Enhanced Filtering for Connectors.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 1bd501e4-be27-1da8-e250-7cc1ac22c8ec
* Version Independent ID: ada2e5a7-646f-3576-5834-4727151b7743
* Content: [Enhanced filtering for connectors in Exchange Online](https://docs.microsoft.com/en-us/exchange/mail-flow-best-practices/use-connectors-to-configure-mail-flow/enhanced-filtering-for-connectors)
* Content Source: [Exchange/ExchangeOnline/mail-flow-best-practices/use-connectors-to-configure-mail-flow/enhanced-filtering-for-connectors.md](https://github.com/MicrosoftDocs/OfficeDocs-Exchange/blob/public/Exchange/ExchangeOnline/mail-flow-best-practices/use-connectors-to-configure-mail-flow/enhanced-filtering-for-connectors.md)
* Service: **exchange-online**
* Technology: ****
* GitHub Login: @JoanneHendrickson
* Microsoft Alias: **jhendr**

**Comment by dariomws** [comment]
Hi @JonesMikael, thank you for your feedback.

If you have a rule that set SCL to -1 for messages that flow through the connector, the messages will bypass/skip spam filtering. Why do you want to enable Enhanced Filtering if there is no spam control? The purpose of Enhanced Filtering is to improve the accuracy of spam controls...

More information:
https://docs.microsoft.com/exchange/security-and-compliance/mail-flow-rules/use-rules-to-set-scl

**Comment by JonesMikael** [comment]
Hi @dariomws and thanks for answering

Here's why I and my colleagues got confused. We've had 3rd party Anti-Spam for years at several customers and they way we have solved this is to put SCL to -1 on all e-mails coming from that IP. Now that we had a new customer with this setup I wanted to investigate Enhanced Filtering so I enabled it (but without SCL to -1) to pilot this. However, we still got many False-Positives in EOP/MDO which confused the customer since they want to have all their filtering in their 3rd party Anti-Spam until we migrate them to EOP/MDE. So I added SCL to -1 but kept Enhanced Filtering enabled. Everything is working as expected and I left it at that because I still think Enhanced Filtering provides value since it preserves IP address and sender information and as a preparation for moving filtering over to EOP/MDE.

It was then I saw that phrase that we **MUST** disable SCL to -1 mail flow rules. I do understand that we **SHOULD** disable those mail flow rules, otherwise Enhanced Filtering doesn't make sense. Maybe add "_If you don't disable the SCL to -1 mail flow rule, no filtering of mails will be performed, hence Enhanced Filtering does not provide as much value as maybe expected_"?
